### ISSUE TITLE
About IDM model

### ISSUE THREAD

**Issue by adoreador** [eclipse-sumo/sumo#14073]
I have a question about IDM model:
In my simulation scenario, I have set up a single-lane road with two vehicles. The simulation time step is 0.1 seconds. For the following vehicle, I've chosen the IDM model with the parameters: minGap="2.5", accel="3", decel="3.4", tau="0.9", stepping="0.1" length="5", desiredMaxSpeed="33.34". When the two vehicles reach a stable state, their speeds are 29.466, and the speed difference is negligible. However, when I retrieve the longitudinal gap using traci.vehicle.getposition, I get a gap of 51.477 - 5 = 46.477. Why isn't the gap equal to minGap + v * tau = 2.5 + 0.9 * 29.466?

In other words, why is the actual “time gap on the lane” not equal to the “desired headway” value in the stable state?
Why don't vehicles proactively accelerate to reduce the following distance and achieve the desired spacing between two cars?

**Comment by m-kro** [comment]
This question can be answered best with an example scenario, as one needs to reach a stable state and you mention a specific IDM parametrisation.

**Comment by adoreador** [comment]
I apologize for any confusion caused by my earlier expression, and I sincerely apologize for that. 
I have rephrased my question. The situation I am facing pertains to a problem encountered while using IDM with a car following model during the simulation process. I am seeking your guidance on this matter, and I haven't resolved the issue yet. 
Once again, I apologize for any confusion caused by my previous miscommunication.

**Comment by m-kro** [comment]
Can you please check if the problem persists if you choose a speed below the edge speed limit for the leader vehicle? (see #14077)

**Comment by adoreador** [comment]
Thanks for the answer!
